README.md

```diff
@@ -420,7 +420,9 @@ Each store has a few events, which you listen:
 
 * `onStart(store, cb)`: first listener was subscribed.
 * `onStop(store, cb)`: last listener was unsubscribed.
-* `onMount(store, cb)`: shortcut to use both `onStart` and `onStop`.
+* `onMount(store, cb)`: shortcut to use both `onStart` and `onStop`. \
```

Comment:
```suggestion
* `onMount(store, cb)`: shortcut to use both `onStart` and `onStop`.
```

and the same on the line below

Reply:
Strange
In VSCode line breaks don't work without the backslash:

<img width="1353" alt="image" src="https://user-images.githubusercontent.com/31626879/178355920-a0d0f3be-aed7-4a1b-80a3-2571e75aa970.png">

Reply:
Hm. Could be VSCode bug.

After https://github.com/nanostores/nanostores/commit/0c34bad1599bf64715a73dc84dc9a00fbff0320c it looks good https://github.com/nanostores/nanostores#store-events

Reply:
@ai not only VSCode but also GitHub =)

<img width="997" alt="image" src="https://user-images.githubusercontent.com/31626879/178356940-e4098ca7-a555-4513-9612-0faa3d7c719a.png">

Reply:
Looks good. What is the problem?

Reply:
I though you wanted to see 80 col width especially in a view mode.
For now I got you 😅